java/arcs/core/entity/StorageAdapter.kt

```diff
@@ -62,4 +70,7 @@ class ReferenceStorageAdapter<E : Entity>(
     override fun referencableToStorable(referencable: StorageReference): Reference<E> {
         return Reference(entitySpec, referencable)
     }
+
+    // Reference handle expiration is not supported.
+    override fun isExpired(value: Reference<E>): Boolean = false
```

Comment:
add a TODO to support `isExpired` for References? A Reference may expire before the underlying Entity does.

Reply:
Adding a todo, but as you can see from here TTL is now wired in reference handle, and the timestamps are missing from core/entity/reference